advance-exercises/src/components/01-CompoundComponents.js

```diff
@@ -16,70 +9,125 @@
     - Arrow left, arrow up should select the previous option
 */
 
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
 class RadioGroup extends React.Component {
   static propTypes = {
-    // defaultValue: PropTypes.string,                UN-COMMENT THIS LINE
-    children: PropTypes.shape().isRequired,
+    defaultValue: PropTypes.string,
+    children: PropTypes.shape().isRequired
   };
   render() {
-    return (
-      <div>{this.props.children}</div>
-    );
+    return <div>{this.props.children}</div>;
   }
 }
 
 class RadioOption extends React.Component {
   static propTypes = {
-    // value: PropTypes.string,                       UN-COMMENT THIS LINE
-    children: PropTypes.shape().isRequired,
+    value: PropTypes.string,
+    children: PropTypes.shape().isRequired
+  };
+  state = {
+    isSelected: false
+  };
+  handleClick = radio => {
+    console.log(this.props);
```

Comment:
One should be careful with committing codes with `console.log()`. Do you think it is needed in the code?

Reply:
@rakeshtembhurne , this was necessary to have a sense of what's going on with the components.

Reply:
Let me know if you recommend that this should be removed.

Reply:
If the task says to keep it, then it's fine. Otherwise all `console.log()` statements should be removed before committing the code. Also, they should not be passed by `eslint`. Kindly check if linting is being properly done.

Reply:
Noted. Thanks 👍